src/components/LogRow/ResmokeRow/index.tsx

```diff
@@ -18,13 +16,7 @@ const ResmokeRow = forwardRef<any, BaseRowProps>((rowProps, ref) => {
   } = data;
   const { index } = listRowProps;
 
-  const line = processedLines[index];
-
-  if (isCollapsedRow(line)) {
-    return (
-      <CollapsedRow ref={ref} {...listRowProps} numCollapsed={line.length} />
-    );
-  }
+  const line = processedLines[index] as number;
```

Comment:
Since we do this check inside of RowRenderer now i wonder if we can update the type for this so we don't have to do this type assertion.